runtime-config-linux.md

```diff
@@ -144,6 +144,12 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+There are a few facets to this:
+
+* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
```

Comment:
On Tue, Jan 05, 2016 at 12:14:37PM -0800, Vincent Batts wrote:
> As is, this could allow for either, though can you point me to where
> unified-hierarchy is explicitly required for this spec?

The wording (from 429f936f, #137) is:

  The Spec does not support [split hierarchy][2]

And the surrounding discussion [1] was (paraphrased):

  me: cgexec(1) lets you set per-controller paths, do we want that?
  @vishh: Won't this break with unified hierarchy [2]?
  me: I see.  Can we link to those kernel docs?

I could see someone reading that as either:

* “we require a __DEVEL__sane_behavior mount” or
* “you only get to specify one cgroup path, not per-controller paths.
  Whether the runtime implements that via an __DEVEL__sane_behavior
  mount or not is undefined”.

Personally I'm in favor of the latter interpretation, but I think the
current wording could go either way.  In either case, there is only
one name for all controllers, so I feel comfortable saying “the
hierarchy” here.

[1]: https://github.com/opencontainers/specs/pull/137#discussion_r38597501
[2]: https://www.kernel.org/doc/Documentation/cgroups/unified-hierarchy.txt